### DISCUSSION TITLE
Find/Replace

### DISCUSSION THREAD

**Discussion by prgmr-1** [codefori/vscode-ibmi#2162] [IBM i]
Find and Replace with wildcards?  can find but not intuitive on replace.

**Comment by chrjorgensen** [comment]
@prgmr-1 Find and Replace is a builtin function in VS Code - not specific to Code for IBM i.

You can find detailed documentation of Find and Replace in the VS Code documentation:
[https://code.visualstudio.com/docs/editor/codebasics#_find-and-replace](https://code.visualstudio.com/docs/editor/codebasics#_find-and-replace)
If not sufficient, search the web for `replace with wildcards in vs code`. You may have to learn about regular expressions (regex) in the process.

Note: [Search across files](https://code.visualstudio.com/docs/editor/codebasics#_search-across-files) is only for local files, not files in QSYS.LIB or IFS!
